GameServerLib/Logic/GameObjects/Spell.cs

```diff
@@ -730,86 +598,10 @@ public void AddPlaceable(float toX, float toY, string model, string name)
-            scriptEngine.RegisterFunction("addProjectileTarget", this, typeof(Spell).GetMethod("AddProjectileTarget", new[] { typeof(string), typeof(Target), typeof(bool) }));
-            scriptEngine.RegisterFunction("getEffectValue", this, typeof(Spell).GetMethod("getEffectValue", new[] { typeof(int) }));
-            scriptEngine.RegisterFunction("spellAnimation", this, typeof(Spell).GetMethod("spellAnimation", new[] { typeof(string), typeof(Unit) }));
-            scriptEngine.RegisterFunction("setAnimation", this, typeof(Spell).GetMethod("setAnimation", new[] { typeof(string), typeof(string), typeof(Unit) }));
-            scriptEngine.RegisterFunction("resetAnimations", this, typeof(Spell).GetMethod("resetAnimations", new[] { typeof(Unit) }));
-            scriptEngine.RegisterFunction("getOtherSpellLevel", this, typeof(Spell).GetMethod("getOtherSpellLevel", new[] { typeof(short) }));
-            scriptEngine.RegisterFunction("addPlaceable", this, typeof(Spell).GetMethod("AddPlaceable", new[] { typeof(float), typeof(float), typeof(string), typeof(string) }));
-            scriptEngine.RegisterFunction("addProjectileCustom", this, typeof(Spell).GetMethod("AddProjectileCustom", new[] { typeof(string), typeof(float), typeof(float), typeof(float), typeof(float), typeof(bool) }));
-            scriptEngine.RegisterFunction("addProjectileCustomTarget", this, typeof(Spell).GetMethod("AddProjectileCustomTarget", new[] { typeof(string), typeof(float), typeof(float), typeof(Target), typeof(bool) }));
-            scriptEngine.RegisterFunction("setCooldown", this, typeof(Spell).GetMethod("SetCooldown", new[] { typeof(byte), typeof(float) }));
-            scriptEngine.RegisterFunction("lowerCooldown", this, typeof(Spell).GetMethod("LowerCooldown", new[] { typeof(byte), typeof(float) }));
-            scriptEngine.RegisterFunction("addLaser", this, typeof(Spell).GetMethod("AddLaser", new[] { typeof(float), typeof(float), typeof(bool) }));
-            scriptEngine.RegisterFunction("teleport", this, typeof(Spell).GetMethod("Teleport", new[] { typeof(Unit), typeof(float), typeof(float) }));
-
-            //scriptEngine.RegisterFunction("addMovementSpeedBuff", this, typeof(Spell).GetMethod("addMovementSpeedBuff", new Type[] { typeof(Unit), typeof(float), typeof(float) }));
-
-            //public void addMovementSpeedBuff(Unit u, float amount, float duration)
-            //{
-            //    Buff b = new Buff(duration);
-            //    b.setMovementSpeedPercentModifier(amount);
-            //    u.AddBuff(b);
-            //    u.GetStats().addMovementSpeedPercentageModifier(b.getMovementSpeedPercentModifier());
-            //}
-
-            scriptEngine.Load(scriptloc); //todo: abstract class that loads a lua file for any lua
-        }*/
-
         public void Teleport(Unit target, float x, float y)
```

Comment:
This was added by @Maufeat , he didn't see `ApiFunctionManager.TeleportTo()` so it would be cool if you remove this, and from Ezreal E and Flash.